instrumentation/kafka-clients/kafka-clients-0.11/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/kafkaclients/ConsumerRecordsInstrumentation.java

```diff
@@ -13,6 +13,9 @@
 import static net.bytebuddy.matcher.ElementMatchers.takesArguments;
 
 import io.opentelemetry.api.trace.SpanContext;
+import io.opentelemetry.instrumentation.kafka.TracingIterable;
+import io.opentelemetry.instrumentation.kafka.TracingIterator;
+import io.opentelemetry.instrumentation.kafka.TracingList;
```

Comment:
These 3 classes shouldn't really be used outside of the kafka-clients javaagent instrumentation.

Reply:
OK, I'll move them back ...